week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,8 +5,16 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
-
+function tidyUpString(strArr) {
+  let newArr = [];
+  for (i in strArr) {
+    let x = strArr[i].trim();
```

Comment:
This is looking good at the moment Bashir - but you can also use chaining too in order to solve this problem. So you can do something like:
`let x = strArr[i].trim().replace('/','');`
So you can put all the chaining in a single line - but it still does the same thing.